generate.sh

```diff
@@ -207,60 +154,60 @@ then
 fi
 
 echo "-> Dockerfiles + devfiles" >&2
+{
 for ocp_ver in "${ocp_versions[@]}"
 do
-    # Soon this might be able to be simplified, as "binaryless" container, FROM scratch
+    pushd catalog/$ocp_ver
```

Comment:
I took some liberty here to show how to use opm to generate the dockerfile for you, even with the image name variability that we were saddled with. 😞

Reply:
I'm ok with opm generate being used here, although there are some differences in the output that make me uneasy

```diff
diff --git a/catalog/v4.11/Dockerfile b/catalog/v4.11/Dockerfile
index 73e0087..dee47c1 100644
--- a/catalog/v4.11/Dockerfile
+++ b/catalog/v4.11/Dockerfile
@@ -7,7 +7,7 @@ ENTRYPOINT ["/bin/opm"]
 CMD ["serve", "/configs", "--cache-dir=/tmp/cache"]
 
 # Copy declarative config root into image at /configs and pre-populate serve cache
-ADD rhbk-operator /configs/rhbk-operator
+ADD rhbk-operator /configs
 RUN ["/bin/opm", "serve", "/configs", "--cache-dir=/tmp/cache", "--cache-only"]
 
 # Set DC-specific label for the location of the DC root directory
```

Reply:
This results in the generated FBC from (for e.g. `./catalog/v4.13/rhbk-operator/catalog.yaml`) getting set in the image as /configs/catalog.yaml. 
For a single-package contribution this is fine, and only differs by omitting the intermediate $operator_name directory. 
If that's important to you, then we just have to move some elements around.

Reply:
As long as IIB and opm are ok with this, I don't particularly care. I never inspect the index image directly, only through `opm render`

Reply:
Understood.  opm considers the specific filesystem hierarchy as FBC and aggregates it, so location doesn't matter to opm... just whether the FBC follows the schema.